### PR TITLE
Test: Adding integration tests for cv_task_v3

### PR DESCRIPTION
**Pull request by Vibhu-gslab**
## Change Summary

Adding integration tests for cv_task_v3

## Related Issue(s)

Fixes #<ISSUE ID>

## Component(s) name

`arista.cvp.cv_task_v3`

## How to test
molecule converge -s cv_task_v3

### User Checklist

- N/A

### Repository Checklist

- [ ] My code has been rebased from devel before I start
- [ ] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [ ] My change requires a change to the documentation and documentation have been updated accordingly. (check the box if not applicable)
- [ ] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Comment by sugetha24** [comment]
Can we follow the same format as the other molecule test and put the tests in a separate playbook and import them in `converge.yml`?

File: ansible_collections/arista/cvp/molecule/cv_task_v3/converge.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+          state: present
+      register: MOVE_DEVICE_DATA_EXECUTE
+
+    - name: Execute Task
+      arista.cvp.cv_task_v3:
+        tasks:
+          - "{{MOVE_DEVICE_DATA_EXECUTE.devices_moved.taskIds[0]}}"
+      register: EXECUTE_TASK_DATA
+
+    - name: Testing Execute Task
```
Comment (sugetha24) [inline ansible_collections/arista/cvp/molecule/cv_task_v3/converge.yml]: Can we get the task facts from CVP and check if they got executed? and same for cancelled example

Comment (Vibhu-gslab) [reply ansible_collections/arista/cvp/molecule/cv_task_v3/converge.yml]: Will complete this once PR #632 gets merged

**Comment by Vibhu-gslab** [comment]
> Can we follow the same format as the other molecule test and put the tests in a separate playbook and import them in `converge.yml`?

Fixed!

**Review by sugetha24 — APPROVED** [review]

**Review by noredistribution — APPROVED** [review]
tested in ATD, lgtm!

**Review by sugetha24 — APPROVED** [review]

File: ansible_collections/arista/cvp/molecule/cv_task_v3/converge.yml (hunk 1 above)
Comment (Vibhu-gslab) [reply ansible_collections/arista/cvp/molecule/cv_task_v3/converge.yml]: Fixed!

**Review by sugetha24 — APPROVED** [review]

**Pull request merged**